src/styles/blocks/page.css

```diff
@@ -0,0 +1,9 @@
+.page {
+  margin-top: 0;
+  font-family: Avenir, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. Consider adding a more generic family as a fallback, such as 'serif' or 'sans-serif' at the end of your font stack.